pages/event-indexing.md

```diff
@@ -0,0 +1,22 @@
+# Indexing ABCI events
+
+The [`pd`](./node/pd) software emits ABCI events while processing blocks. By default,
+these blocks are stored in CometBFT's key-value database locally, but node operators
```

Comment:
```suggestion
The [`pd`](./node/pd) software emits ABCI events while processing blocks.
By default, these blocks are stored in CometBFT's key-value database locally, but node operators
```

nit: extremely pedantic, and non-blocking, but I'd be remiss if I didn't point out a "cool markdown tip", which is that by making it so that sentences always start on a new line, reviewing subsequent diffs are much easier.

Reply:
This is a sick tip. Generally I veer toward shorter lines for plaintext editing, but I acknowledge I'm in a minority in my aversion to textwrapping. Great suggestion, will implement!